app/services/return-requirements/delete-note.service.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+/**
+ * Orchestrates deleting the notes data for `/return-requirements/{sessionId}/check-your-answers` page
+ * @module DeleteNoteService
+ */
+
+const SessionModel = require('../../models/session.model.js')
+
+/**
+ *
+ * It first retrieves the session instance for the returns requirements journey in progress.
+ *
+ * Then it removes the notes data from the session.
+ *
+ * @param {string} sessionId - The id of the current session
+ * @param {Object} yar - The Hapi `request.yar` session manager passed on by the controller
+ *
+ * @returns {Promise<Object>} The page data for the check-your-answers page
+ */
+async function go (sessionId, yar) {
+  const session = await SessionModel.query().findById(sessionId)
+  const notification = {
+    title: 'Removed',
+    text: 'Note removed'
+  }
+
+  await _save(session)
+
+  yar.flash('notification', notification)
+
+  return notification
+}
+
+async function _save (session) {
+  const currentData = session.data
+
+  currentData.note = ''
```

Comment:
If we want to delete the note we ideally want to put it back to the state it was before we added one in the first place. If that is the case I think you need to call this instead.

```suggestion
  delete currentData.note
```